src/content/content.component.ts

```diff
@@ -17,97 +18,58 @@ import { BUTTON_TYPE } from '@shared-components/button/button.component';
+      .subscribe(allMovies => this.allMovies = allMovies);
+    this.movieFacade.managedMovies$
+      .pipe(untilDestroyed(this))
+      .subscribe(movies => this.managedMovies = movies);
   }
 
   get sortedMoviesList(): Movie[] {
     let sliceMovies = this.slicePage(this.managedMovies, this.page);
     if (sliceMovies.length === 0) {
       const newPageIndex = this.page.pageIndex !== 0 ? this.page.pageIndex - 1 : 0;
-      this.page = { ...this.page, ...{ pageIndex: newPageIndex } };
+      this.page = {...this.page, ...{pageIndex: newPageIndex}};
       sliceMovies = this.slicePage(this.managedMovies, this.page);
     }
     return sliceMovies;
   }
 
-  get allMovies(): Movie[] {
-    const all = this.moviesService.allMovies;
-    const onlyFavorite = all.filter((movie: Movie) => movie.favorite === true);
-    return this.onlyFavoriteMovie ? onlyFavorite : all;
-  }
-
   get getListClass(): string {
     return this.type === DISPLAY_TYPE.LIST ? 'column' : 'wrap';
   }
 
-  handleApiResponse(type: VIDEO_WEBSITE, idVideo: string): void {
+  handleApiResponse(type: VIDEO_WEBSITE, id: string): void {
     if (type === VIDEO_WEBSITE.VIMEO) {
-      this.apiService.fetchVimeoApi(idVideo)
-        .subscribe((res: VimeoApiResponse) => {
-          const movie: Movie = {
-            movieId: idVideo,
-            imageUrl: res.pictures.sizes[0].link,
-            title: res.name,
-            viewCount: '',
-            publishedAt: res.created_time,
-            url: `https://player.vimeo.com/video/${idVideo}`,
-            favorite: false
-          };
-          this.moviesService.addMovie(movie);
-        },
-          (err) => {
-            this.openSnackBar('To jest niepoprawny link');
-            console.error('Handle error from Vimeo', err);
-          }
-        );
+      this.movieFacade.loadMovieFromVimeo(id);
     }
     if (type === VIDEO_WEBSITE.YOUTUBE) {
-      this.apiService.fetchYoutubeApi(idVideo)
-        .subscribe((res: YoutubeApiResponse) => {
-          if (res.items.length === 0) {
-            this.openSnackBar('To jest niepoprawny link');
-            return;
-          }
-          const { id, snippet, statistics } = res.items[0];
-          const movie: Movie = {
-            movieId: id,
-            imageUrl: snippet.thumbnails.default.url,
-            title: snippet.title,
-            viewCount: statistics.viewCount,
-            publishedAt: snippet.publishedAt,
-            url: `https://www.youtube.com/embed/${id}`,
-            favorite: false
-          };
-          this.moviesService.addMovie(movie);
-        },
-          (err) => console.error('Handle error from Youtube', err)
-        );
+      this.movieFacade.loadMovieFromYoutube(id);
     }
   }
 
   handleValue(valueFromInput: any): void {
     this.value = valueFromInput;
-    const { idVideo, videoWebsite } = extractIdAndWebsiteType(valueFromInput);
+    const {idVideo, videoWebsite} = extractIdAndWebsiteType(valueFromInput);
```

Comment:
te spacje według mnie robią robotę, czytelniej
 
```suggestion
    const { idVideo, videoWebsite } = extractIdAndWebsiteType(valueFromInput);
```